internal/provider/provider_test.go

```diff
@@ -22,7 +22,7 @@ func testAccPreCheck(t *testing.T) {
 	// function.
 
 	// Check environmental variables
-	ev := []string{"MERAKI_DASHBOARD_API_KEY"}
+	ev := []string{"MERAKI_DASHBOARD_API_KEY, TF_ACC_MERAKI_MX_LICENCE, TF_ACC_MERAKI_MX_SERIAL, TF_ACC_MERAKI_ORDER_NUMBER"}
```

Comment:
Seems like the same change as in the add devices resource so this should hopefully not cause a merge conflict.

Reply:
added them to secrets